.github/workflows/bootstrap.yml

```diff
@@ -142,10 +148,11 @@ jobs:
         run: |
           response=$(curl https://sentry.io/api/0/teams/ministryofjustice/probation-integration/projects/ --fail \
             -H "Authorization: Bearer $SENTRY_AUTH_TOKEN" \
-            -H 'Content-Type: application/json' --data '{"name":"${{ inputs.project_name }}"}')
+            -H 'Content-Type: application/json' --data '{"name":"$project_name"}')
```

Comment:
I don't think this one will work as it's in single quotes - might need jq, or piling on more quotes...